### PR TITLE
Text extraction

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/933

Works with:
- https://github.com/Islandora-CLAW/islandora/pull/163
- https://github.com/Islandora-CLAW/Crayfish/pull/77

# What does this Pull Request do?

Config to trigger OCR derivatives using `islandora_text_extraction`

# What's new?
Re-worked contexts and moves the field storage for the 'weight' field over to `islandora_core_feature`.

# How should this be tested?

Test PR coming soon

# Interested parties
@Islandora-CLAW/committers @dbernstein @ajstanley

### CODE REVIEW

File: config/install/context.context.pages.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+label: Pages
+group: Islandora
+description: 'Context for page objects'
+requireAllConditions: true
+disabled: false
+conditions:
+  parent_node_has_term:
+    id: parent_node_has_term
+    negate: 0
+    uuid: 508b74f7-6ed5-4439-a170-4b014bbca4ce
```
Comment (seth-shaw-unlv) [inline config/install/context.context.pages.yml]: These UUIDs shouldn't be here...

File: config/install/context.context.pages.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    uri: 'http://id.loc.gov/ontologies/bibframe/part'
+    context_mapping:
+      media: '@islandora.media_route_context_provider:media'
+  media_has_term:
+    id: media_has_term
+    negate: 0
+    uuid: 68e99964-d23d-4a6e-a27f-dd9d0b34db93
```
Comment (seth-shaw-unlv) [inline config/install/context.context.pages.yml]: UUIDs cause trouble.

**Comment by dannylamb** [comment]
@seth-shaw-unlv Ok, I squashed all my previous commits into one and then tacked the UUID removal commit afterwards.  When merging, please _do not squash and merge_ so that last commit can be reverted separately in case something blows up when testing with a re-provision.  Everything will probably be fine, but I'm paranoid like that.

**Comment by dannylamb** [comment]
FYI I used this command to destroy the UUIDs:

```
$ sed -i '/uuid:/d' config/install/*.yml
```

**Comment by seth-shaw-unlv** [comment]
Don't know why Travis couldn't find the storage config for field_weight in islandora_core_feature... it is there...

**Pull request merged**